discovery-infra/test_infra/controllers/node_controllers/qe_vm_controler.py

```diff
@@ -14,3 +17,35 @@ def format_node_disk(self, node_name):
 
     def get_ingress_and_api_vips(self):
         return {"api_vip": "192.168.123.5", "ingress_vip": "192.168.123.10"}
+
+    def set_boot_order_to_node(self, node_name, cd_first=False):
```

Comment:
move it to libvirt_controller and lets it to interface.
Add function to host and lets use it from host object. 
I hope to move to use host in tests and not node_controller